internal/isoeditor/rhcos.go

```diff
@@ -382,6 +404,66 @@ func IgnitionImageArchive(ignitionConfig string) ([]byte, error) {
 	return compressedBuffer.Bytes(), nil
 }
 
+func readHeader(isoPath string) (*OffsetInfo, *OffsetInfo, error) {
+	iso, err := os.OpenFile(isoPath, os.O_RDONLY, 0o664)
+	if err != nil {
+		return nil, nil, err
+	}
+	defer iso.Close()
+
+	// Read header
+	header := make([]byte, headerLength)
```

Comment:
Why read the entire header? Couldn't we read just the 48 bytes we care about?

Reply:
ok, used ReadAt to read only the relevant 48 bytes.